corehq/apps/locations/tests/test_location_expressions.py

```diff
@@ -69,59 +69,71 @@ def test_location_not_found(self):
         self._check_expression(doc, None)
 
 
+def setup_module():
+    domain = "a-song-of-ice-and-fire"
+    domain_obj = create_domain(domain)
+    continent_location_type = LocationType(
+        domain=domain,
+        name="continent",
+        code="continent",
+    )
+    continent_location_type.save()
+    kingdom_location_type = LocationType(
+        domain=domain,
+        name="kingdom",
+        code="kingdom",
+        parent_type=continent_location_type,
+    )
+    kingdom_location_type.save()
+    city_location_type = LocationType(
+        domain=domain,
+        name="city",
+        code="city",
+        parent_type=kingdom_location_type,
+    )
+    city_location_type.save()
+
+    parent = SQLLocation(
+        domain=domain,
+        name="Westeros",
+        location_type=continent_location_type,
+        site_code="westeros",
+    )
+    parent.save()
+    child = SQLLocation(
+        domain=domain,
+        name="The North",
+        location_type=kingdom_location_type,
+        parent=parent,
+        site_code="the_north",
+    )
+    child.save()
+    grandchild = SQLLocation(
+        domain=domain,
+        name="Winterfell",
+        location_type=city_location_type,
+        parent=child,
+        site_code="winterfell",
+    )
+    grandchild.save()
+
+    globals()["domain_obj"] = domain_obj
+    globals()["domain"] = domain
+    globals()["parent"] = parent
+    globals()["child"] = child
+    globals()["grandchild"] = grandchild
```

Comment:
I'm abandoning this approach.

This `setup_module()` function seems to get called during `./manage.py test -sx corehq.apps.locations.tests.test_location_expressions --attr=sql_backend`, even though this runs 0 tests. The test runner uses a `ErrorOnDbAccessContext` context for this test run which replaces all dbs with mocks, and therefore this function throws exceptions.
Don't have the time at the moment to figure out why this function is being called when there are no tests, so reverting for now.